pkg/sanity/sanity.go

```diff
@@ -69,10 +69,12 @@ type Config struct {
 	TestVolumeSize int64
 
 	// Target size for ExpandVolume requests. If not specified it defaults to TestVolumeSize + 1 GB
-	TestVolumeExpandSize      int64
-	TestVolumeParametersFile  string
-	TestVolumeParameters      map[string]string
-	TestNodeVolumeAttachLimit bool
+	TestVolumeExpandSize       int64
+	TestVolumeParametersFile   string
+	TestVolumeParameters       map[string]string
+	TestSnapshotParametersFile string
```

Comment:
Can you add the new fields in a separate block (i.e. don't mix with `TestVolumeExpand*`) and add documentation? I know it's missing also for the other fields, but that's not a good example to follow :man_shrugging: